services/frontend-v3/src/i18n/en_CA.json

```diff
@@ -251,6 +252,7 @@
   "profile.rules.max.1500": "Max length 1500 characters",
   "profile.rules.max.50": "Max length 50 characters",
   "profile.rules.max.60": "Max length 60 characters",
+  "profile.rules.max": "Max length {max} characters",
```

Comment:
I'm not sure if this is out of scope for this PR, if you introduce this, should we remove the other "profile.rules.max.XX"?

Reply:
I want to remove it but let me make another PR for it so i can test it better and isolate it